docs/develop/develop-add-ins-visual-studio.md

```diff
@@ -0,0 +1,56 @@
+
+> [!NOTE]
+> As an alternative to using Visual Studio, you may choose to use the Yeoman generator for Office Add-ins and VS Code to create an Office Add-in. For more information about this choice, see [Creating an Office Add-in](../overview/office-add-ins-fundamentals.md#creating-an-office-add-in).
+
+## Create the add-in project using Visual Studio
+
+Visual Studio can be used to create Office Add-ins for Excel, Outlook, Word, and PowerPoint. An Office Add-in project gets created as part of a Visual Studio solution and uses HTML, CSS, and JavaScript. To create an Office Add-in with Visual Studio, follow instructions in the quick start that corresponds to the add-in you'd like to create:
+
+- [Excel quick start](../quickstarts/excel-quickstart-jquery.md?tabs=visualstudio)
+- [Outlook quick start](/outlook/add-ins/quick-start?context=office/dev/add-ins/context&tabs=visualstudio)
+- [Word quick start](../quickstarts/word-quickstart.md?tabs=visualstudio)
+- [PowerPoint quick start](../quickstarts/powerpoint-quickstart.md?tabs=visualstudio)
+
+Visual Studio does not support creating Office Add-ins for OneNote or Project. To create Office Add-ins for either of these hosts, you'll need to use the Yeoman generator for Office Add-ins, as described in the [OneNote quick start](../quickstarts/onenote-quickstart.md) or the [Project quick start](../quickstarts/project-quickstart.md).
+
+## Develop the add-in using Visual Studio
+
+Visual Studio creates a basic add-in with limited functionality. You can customize the add-in by editing the [manifest](add-in-manifests.md), HTML, JavaScript, and CSS files in Visual Studio. For a high-level description of the project structure and files in the add-in project that Visual Studio creates, see the Visual Studio guidance within the quick start that you completed to create your add-in. 
+
+> [!TIP]
+> Because an Office Add-in is a web application, you'll need at least basic web development skills to customize your add-in. If you're new to JavaScript, we recommend reviewing the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
+
+To customize your add-in, you'll need to understand concepts described in the [Core concepts > Develop](develop-overview.md) area of this documentation, as well as concepts described in the host-specific area of documentation that corresponds to the add-in you're building (for example, [Excel](../excel/index.md)). 
+
+## Test and debug the add-in
+
+Methods for testing, debugging, and troubleshooting Office Add-ins vary by platform. For more information, see [Debug Office Add-ins in Visual Studio](debug-office-add-ins-in-visual-studio.md) and [Test and debug Office Add-ins](../testing/test-debug-office-add-ins.md).
+
+## Publish the add-in
```

Comment:
I like this section, I feel like it is a lot more clear than previous iterations on this topic